proposal/model-registry.md

```diff
@@ -0,0 +1,62 @@
+It is an important component in the life cycle of AI/ML models. 
+A model registry provides a central repository for model developers to store and manage models, versions, and model metadata. 
+It fills a gap between model experimentation and model serving activities. It provides a central interface for 
+stakeholders in the ML life cycle to collaborate on models in a secure fashion.
+
+During the development and deployment life cycle of an ML model, various artifacts and metadata are created or associated with the ML model.
+For example, a *Data Scientist* may create training data, hyperparameters, metrics during development. 
+During deployment, an *MLOps Engineer* may work with a versioned Model from the repository for model-serving to provide Inferencing on the Model. 
+Pre-trained ML models such as Foundation Models when used directly for model serving will include metadata about their artifacts and hyperparameters. 
+Model serving will generate metrics from inference. 
+Execution of actions and events performed during the model's lifecycle, for example in a pipeline that executes various steps to process and transform data, can also be stored as metadata. 
+
+There needs to be a central repository for all such metadata so that it can provide visibility across the entire lifecycle of the model. 
+It should be able to provide lineage and provenance for any artifact or metadata related to an ML model. 
+
+**Design Goals**
+
+* Model Registry should be a passive repository which receives and stores ML model related metadata pushed to the registry.
+* The registry may have simple rules for validating metadata, but it does not take any actions or perform any orchestration based on this metadata. 
+* The metadata can be pushed to the repository using client libraries in code written by ML users or automatically generated by other ODH components in the platform. 
+For example, metadata about the deployment status of an ML Model. 
+* The Model Registry should also support Kubernetes native deployment just like other Open Data Hub Components. 
+
+**User Use Cases**
+- *Data Scientists* need access to a Model Registry to store information about Experiments and Runs for Tracking and reproducibility. 
+- *Data Scientists* can register versions of ML models in the Model Registry to share with other *Data Scientists* and *MLOps Engineers*. 
+- *Data Scientists* and *MLOps Engineers* can design and implement kubeflow pipelines that register their artifacts and executions in the Model Registry. 
+- *Data Scientists* and *MLOps Engineers* can browse the Model Registry to deploy ML models to various environments (e.g. deployment stages like *Stage* and *Production*, locations like *East, West* or specific devices). 
+
+## Architecture
+The high level architecture for an OpenDatahub Model Registry is depicted below.
+
+![image](https://github.com/opendatahub-io/opendatahub-community/assets/181691/62b9ea30-a1d2-47d6-8f97-c4bfa5ba8a88)
```

Comment:
Can you replace it with latest version from here https://miro.com/app/board/uXjVMi2jnrI=/ ?

Reply:
Ramesh has another PR with updates the diagrams that needs to be rebased and merged.